Zend/zend_vm_gen.php

```diff
@@ -19,7 +19,7 @@
 	 $Id$
 */
 
-$header_text = <<< DATA
+const HEADER_TEXT = <<< DATA
```

Comment:
Yeah, that's why I said, "latest stable version" and not, "compiled php binary from master". ;)